### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by loysollivier**
Add and update requirements file

Signed-off-by: Loys Ollivier <lollivier@baylibre.com>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
+# Next dependencies are Python dependencies
 Flask==0.10.1
+Flask-WTF==0.11
+Flask-Cache==0.13.1
 redis==2.10.5
-hiredis
-requests
-simplejson
+hiredis==0.2.0
+requests==2.9.1
```
Comment (MiloCasagrande) [inline requirements.txt]: Production server is actually using 2.18.1 for requests.

File: requirements.txt [hunk 2]
```diff
@@ -1,8 +1,13 @@
-Flask-Cache
-Flask-WTF==0.11
+python==2.7.12
+python-dev==2.7
```
Comment (MiloCasagrande) [inline requirements.txt]: We are using 2.7.9 in production, on a Jessie install.
Maybe `>=2.7.9,<=2.7.12`?

Where does the 2.7.12 come from?

Comment (loysollivier) [reply requirements.txt]: As there was no version number prior to this commit I just used the installation default ones and checked that it worked. If you have any version number already supported I'd be glad to have them :)

File: requirements.txt (hunk 1 above)
Comment (loysollivier) [reply requirements.txt]: OK, I'll contact you concerning the version numbers

**Pull request merged**